CHANGELOG.md

```diff
@@ -1,4 +1,30 @@
 ## 2.12.0 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-5.3.1486** 
```

Comment:
This version will change; another patch is expected roll out by tomorrow

Reply:
Just say 5.3.x if it is compatible with all of them